### DISCUSSION TITLE
Do we still want/need a rules page?

### DISCUSSION THREAD

**Discussion by aedwardg** [CodecademyCommunity/CodecademyCommunity.github.io#12] [Q&A] [answered]
We discussed having a dedicated Rules page on this site for a more in-depth version of the server rules and possibly some examples of rule-breaking behavior.  I also included it in my original website brainstorm here: 
https://discord.com/channels/810930683732033597/848577858388492298/848648695259988018

Since then, we've updated the rules to be fairly lengthy on Discord.  So, the question now is: do we still want or need this rules page?

**Comment by Victoria-DR** [comment, marked as answer]
I think the rules as we have them now (long form on Discord) are enough. They are lengthy but an issue with having a short format on Discord and the full set of rules on the site means that more people are going to skip over reading the rules since they aren't as easily accessible.
